### DISCUSSION TITLE
Import .oim file?

### DISCUSSION THREAD

**Discussion by rpshi** [mtex-toolbox/mtex#796] [Ask Anything]
Can MTEX import .oim file? or .ang is required? 

can .oim file be converted to be .ang file

Thanks,

**Comment by lfgmorales** [comment]
Hi

You cannot import oim files in MTEX, you can either use your *.osc file or export it using OIM analysis as *.ang. Please be aware that some versions of *.osc will also not work, while *.ang files are generally not a problem

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 20 Feb 2021, at 03:49, rpshi <notifications@github.com> wrote:

﻿

Can MTEX import .oim file? or .ang is required?

can .oim file be converted to be .ang file

Thanks,

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/796>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN3O4WJMICDVTHHHSXTS74PMVANCNFSM4X5MFK3Q>.

[ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/796", "url": "https://github.com/mtex-toolbox/mtex/discussions/796", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
